pkg/skaffold/gcp/client.go

```diff
@@ -17,60 +17,22 @@ limitations under the License.
 package gcp
 
 import (
-	"context"
-	"sync"
-
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
-
-	cstorage "cloud.google.com/go/storage"
-	"github.com/pkg/errors"
-	cloudbuild "google.golang.org/api/cloudbuild/v1"
 	"google.golang.org/api/option"
-)
 
-var (
-	cbclient *cloudbuild.Service
-	cbOnce   sync.Once
-	cbErr    error
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
 )
 
-// CloudBuildClient returns an authenticated client for interacting with
-// the Google Cloud Build API. This client is created once and cached
-// for repeated use in Skaffold.
-func CloudBuildClient() (*cloudbuild.Service, error) {
-	cbOnce.Do(func() {
-		var options []option.ClientOption
-		var err error
-		creds, cErr := activeUserCredentials()
```

Comment:
why are we removing this? This is an important piece of logic that we implemented to fix https://github.com/GoogleContainerTools/skaffold/issues/1966 - unless I'm missing something and the client libraries include this logic now?

Reply:
This is not removed, just moved to `ClientOptions()`

Reply:
gotcha, I missed that